salt/modules/yumpkg.py

```diff
@@ -2885,3 +2922,60 @@ def diff(*paths):
                     local_pkgs[pkg]['path'], path) or 'Unchanged'
 
     return ret
+
+
+def _get_patches(installed_only=None):
```

Comment:
If we're passing ``installed_only=True`` to enable it, shouldn't the default value be ``False`` and not ``None``? I get that it will work either way, but using ``False`` as a default just feels better here.